static/src/js/components/TemporalDisplay/__tests__/TemporalSelectionDropdown.test.js

```diff
@@ -180,15 +187,22 @@ describe('TemporalSelectionDropdown component', () => {
     const validEndDate = '2019-03-30T00:00:00.000Z'
     const validStartDate = '2019-03-29T00:00:00.000Z'
 
-    fireEvent.click(screen.getByRole('button'))
+    const expectedEndDate = '2019-03-30T23:59:59.999Z'
 
-    const inputs = screen.getAllByRole('textbox')
+    await act(async () => {
+      await user.click(screen.getByRole('button'))
```

Comment:
```suggestion
      await user.click(screen.getByRole('button', {name: '<what-ever-name-is>))
```
nitpik; makes it easier to read if we hav a filter on the button